package.json

```diff
@@ -10,8 +10,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint:js": "eslint --ignore-path .gitignore --ext .js,.jsx i",
-    "lint:css": "prettier **/*.{json,scss,css}",
+    "lint:js": "eslint --ignore-path .gitignore --ext .jsx --ext .js .",
+    "lint:css": "prettier '**/*.{json,scss,css}' --ignore-path .gitignore",
```

Comment:
I suggest to write initial config options first and put file path the last but it does not really matter. Just easier to read and consistent with the command above.